### PR TITLE
Private chat links

### PR DESCRIPTION
**Pull request by uburuntu**
# Description

Support links for non-public chats from this update:
https://telegram.org/blog/archive-and-new-design#easier-forwarding-links-to-messages-online-badges

For example: `https://t.me/c/1122334455/1`

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Via package tests

**Test Configuration**:
* Operating System: Windows 10
* Python version: 3.7

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: aiogram/types/chat.py [hunk 1]
```diff
@@ -63,6 +63,16 @@ def user_url(self):
...
 
+    @property
+    def shifted_id(self) -> int:
+        """
+        Get shifted id of chat, e.g. for private links
+
+        For example: -1001122334455 -> 1122334455
+        """
+        shift = -1000000000000
+        return shift - self.id
```
Comment (JrooTJunior) [inline aiogram/types/chat.py]: Private chats is also should have shifted id?

Comment (uburuntu) [reply aiogram/types/chat.py]: Yes, you can double-click (or long tap) on message and click "Copy link"

Comment (uburuntu) [reply aiogram/types/chat.py]: Hmm, I thought you were talking about closed chats. So private chat with bot doesnt have shifted_id, I will add check for it.

**Comment by JrooTJunior** [comment]
@uburuntu, do you have any news about this PR?

**Comment by uburuntu** [comment]
Done :star2:

File: aiogram/types/chat.py (hunk 1 above)
Comment (MasterGroosha) [reply aiogram/types/chat.py]: I'm sure you can use `-1_000_000_000_000` for readability without breaking anything.

Comment (uburuntu) [reply aiogram/types/chat.py]: Cool, but for what? It is internal numbering of Telegram, not for calculation or anything else

**Comment by uburuntu** [comment]
@MasterGroosha review and merge pls

**Comment by MasterGroosha** [comment]
I trust @JrooTJunior since he's the creator of aiogram

**Pull request merged**